go.mod

```diff
@@ -107,3 +107,7 @@ require (
 )
 
 go 1.18
+
+replace github.com/codeready-toolchain/api => github.com/mfrancisc/api v0.0.0-20221204201849-a5b2363b5d69
```

Comment:
reminder

Reply:
fixed in: [3de7b336108](https://github.com/codeready-toolchain/host-operator/pull/718/commits/3de7b336108d279b451f8c5b1b10bc95d8c66594)